routing/fake_ending.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "routing/road_graph.hpp"
+#include "routing/segment.hpp"
+
+#include "geometry/point2d.hpp"
+
+#include <vector>
+
+namespace routing
+{
+class WorldGraph;
+class EdgeEstimator;
+class IndexGraph;
+
+struct Projection final
+{
+  Segment m_segment;
+  bool m_isOneWay;
```

Comment:
Please, add default initialization for the POD fields.

Reply:
done